src/java.base/share/classes/java/io/CharArrayReader.java

```diff
@@ -146,16 +146,11 @@ public int read(char b[], int off, int len) throws IOException {
     }
 
     /**
-     * Skips characters.  Returns the number of characters that were skipped.
-     *
+     * {@inheritDoc}
```

Comment:
Using inheritDoc for the exceptions adds several new and irrelevant impossible causes to the javadoc.
" This method will block until some characters are available", etc.
It also drops the existing "If the stream is closed" condition on @ throws IOException.

Reply:
As above for `StringReader`.